library/Vanilla/Models/FsThemeProvider.php

```diff
@@ -238,14 +258,15 @@ public function getAssetData($themeKey, string $assetKey): string {
     }
 
     /**
-     * Get theme asset by assetKey.
+     * Get theme assets by by themeID.
+     *
+     * @param string $themeID
      *
-     * @param string $id
      * @return mixed
      * @throws NotFoundException Throws an exception if asset not found.
      */
-    private function getAssets(string $id): array {
-        $theme = $this->getThemeByName($id);
+    private function getAllDefinedAssets(string $themeID): array {
```

Comment:
Why do we need to rename function? I don't see any profit.

Reply:
Done